rust-examples/operators.rs

```diff
@@ -120,6 +120,9 @@ mod wasm {
     }
 }
 
+// This function will be available to other Modules.
+// And can also be called by the interpreter at Module start (binaryen-shell -e test) 
+#[wasm_export]
```

Comment:
In the end the visibility of symbols in wasm should be controlled the same way as in Rust. It's not as simple as just exporting everything `pub` though. You have to export everything 'reachable'. There's definitely some data structure in the compiler that contains this info already. I don't mind doing this for now, but it'll need to be fixed at some point.